### PR TITLE
Add Neofetch CMD (sssh! its a secret :P)

### PR DESCRIPTION
**Pull request by CaenJones**
Hi Ishan!

### CODE REVIEW

File: commands.go [hunk 1]
```diff
@@ -74,6 +75,9 @@ var (
 		{"colour", colorCMD, "???", "This is an alias of color"}, // appease the british
 		{":q", exitCMD, "", "This is an alias of exit"},          // appease the Vim user
 		{":wq", exitCMD, "", "This is an alias of exit"},         // appease the Vim user, that wants to save
+		{"finger", lsCMD, "???", "This is an alias of id"}, // appease the Linux / Unix user
```
Comment (quackduck) [inline commands.go:78]: did you mean to make this an alias to idCMD?

File: commands.go [hunk 2]
```diff
@@ -74,6 +75,9 @@ var (
...
+		{"del", rmCMD, "???", "This is an alias of rm"}, //apease the MS-DOS user :)
```
Comment (quackduck) [inline commands.go:79]:
```suggestion
```

~~devzat does not appease the ms-dos user. we shun the ms-dos user. mwahaha~~

File: commands.go [hunk 3]
```diff
@@ -74,6 +75,9 @@ var (
...
+		{"neofetch", neofetchCMD, "???", "???"}, //apease the Arch user (mostly)
```
Comment (quackduck) [inline commands.go:80]: neofetch!!! great idea!!! let's make it super good.

File: discord.go [hunk 4]
```diff
@@ -138,6 +138,13 @@ func discordMessageHandler(_ *discordgo.Session, m *discordgo.MessageCreate) {
 	if m == nil || m.Author == nil || m.Author.Bot || m.ChannelID != Integrations.Discord.ChannelID { // ignore self and other channels
 		return
 	}
+
+	// Process only if the message starts with "cd " or "cd\n"
+	msgContent := strings.TrimSpace(m.ContentWithMentionsReplaced())
+	if strings.HasPrefix(msgContent, "cd ") || msgContent == "cd" {
+		return
+	}
+
```
Comment (quackduck) [inline discord.go:147]: Please make a pr only about one thing at a time!

File: discord.go [hunk 5]
```diff
@@ -146,7 +153,6 @@ func discordMessageHandler(_ *discordgo.Session, m *discordgo.MessageCreate) {
 	}
 	DiscordUser.Name = Magenta.Paint(Integrations.Discord.Prefix+" ") + (Styles[int(i)%len(Styles)]).apply(name)
 
-	msgContent := strings.TrimSpace(m.ContentWithMentionsReplaced())
```
Comment (quackduck) [inline discord.go:149]: Why was this deleted!?

File: commands.go (hunk 1 above)
Comment (CaenJones) [reply commands.go:78]: yes

**Comment by quackduck** [comment]
[macos.txt](https://github.com/quackduck/devzat/files/12135568/nfetch.txt)
[endeavouros.txt](https://github.com/quackduck/devzat/files/12135569/send-to-ishan.txt)

here's two sample neofetch outputs for us to emulate

**Comment by CaenJones** [comment]
agreed, will work on it tomorow

File: commands.go [hunk 6]
```diff
@@ -811,3 +813,32 @@ func uptimeCMD(rest string, u *User) {
...
+
+// Then you can add the "neofetch" command to the struct like this:
+var commands = []Command{
+	{
+		Name:        "neofetch",
+		HandlerFunc: neofetchCMD,
+		Description: "apease the Arch user (mostly)",
+	},
+
+}
```
Comment (quackduck) [inline commands.go:844]:
```suggestion
```

**Comment by CaenJones** [comment]
Thank you

**Comment by CaenJones** [comment]
To see the normal looking ascii art: go to: https://github.com/CaenJones/devzat/blob/main/devzat-data/neofetch.txt
(fits on terminal BTW)
